lab-caleb/lib/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.products = {};
+exports.products.gfxcard = [];
+exports.products.keyboard = [];
+
+
+exports.Product = function(name, price, sku){
+  this.addToProducts = function(){
+    exports.products[this.category].push(this);
+  };
+  this.name = name;
+  this.price = price;
+  this.sku = sku;
+};
+
+exports.Card = function(name, price, sku, architecture, category){
+  exports.Product.call(this, name, price, sku);
+  this.architecture = architecture;
+  this.category = category;
+};
+
+exports.Keyboard = function(name, price, sku, backlighting, category){
+  exports.Product.call(this, name, price, sku);
+  this.backlighting = backlighting;
+  this.category = category;
+};
+
+//to test call find a way to describe it, and verify that it has the properties that the product constructor gives.
+//gfxcard
```

Comment:
Won't be docking points for this yet, but get in the habit of managing comments on a separate branch for your own reference and submit a clean branch to Canvas.